backend/user/views.py

```diff
@@ -150,6 +156,70 @@ def delete(self, request, admin_id):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class SaveUnitApiView(APIView):
+    """
+    An endpoint to handle saving a vehicle to a bidder's saved vehicle list,
+    as well as retrieving a list of all saved vehicles
+    """
+
+    def get(self, request, **kwargs):
+        # Replace this line to obtain user id once authentication has been implemented
+        bidder_id = kwargs.get("bidder_id")
+        bidder = get_object_or_404(Bidder, id=bidder_id)
+
+        saved_units = SavedUnits.objects.filter(bidder_id=bidder)
```

Comment:
Filter by `bidder_id` and `auction_id`. We want the saved vehicles for a given auction from that bidder